database_handling.py

```diff
@@ -24,8 +21,32 @@
     Column('part_color', String(50)),
     Column('part_quantity', String(50))
 )
+minifigure_prices = Table(
+    'minifigure_prices', meta, 
+    Column('id', String(50), primary_key = True ),
+    Column('minifigure_id', String(50)),
+    Column('capture_date', Date),
+    Column('unit_price', Float),
+    Column('quantity', Integer)
+)
 meta.create_all(engine)
 
-conn = engine.connect()
-ins = minifigure_parts.insert().values(key = 'alp0011346565', minifigure_id= 'alp001')
-result = conn.execute(ins)
\ No newline at end of file
+def insert_to_table(list_of_dicts, tablename):
+    if len(list_of_dicts) > 0:
```

Comment:
you can use an early return here to reduce nesting
```suggestion
    if len(list_of_dicts) == 0:
      return
```